### DISCUSSION TITLE
why editor can't go full screen?

### DISCUSSION THREAD

**Discussion by xtheeq** [wezterm/wezterm#5905] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!
I'm using helix rn,

and while having the config as
``` config.window_padding = {top = 0, bottom =0, right = 0, left = 0```

![image](https://github.com/user-attachments/assets/69bb1cd5-c2a3-4bf8-94eb-8edd8bb65b8a)
how to fill the full screen? the white part are not filled 

i get the most centered one using
``` config.window_padding = {top = 3, bottom =10, right = 0, left = 0```

![image](https://github.com/user-attachments/assets/439ad746-42da-40a0-baba-3db87c3c4424)

**Comment by MartyMcFlyInTheSky** [comment]
IIRC you can just press WindowsKey + Up to maximize a window no? Or do you insist on a configurable solution?

**Reply by xtheeq** [reply]
I'm not asking about making the entire WezTerm window full screen.

My issue is that Helix, which I use inside WezTerm, doesn't fully occupy the available WezTerm space even though I've set the padding to 0.

I've chosen a white theme for WezTerm to distinguish between the editor theme and the remaining WezTerm space.

I need Helix to utilize the white space within WezTerm.
